app/assets/stylesheets/application.css.scss

```diff
@@ -8,21 +8,28 @@
  * You're free to add application-wide styles to this file and they'll appear at the top of the
  * compiled file, but it's generally better to create a new file per style scope.
  *
- *= require scaffolds
- *= require_self
- *= require home
- *= require loans
- *= require users
- *= require media
- *= require embedded
- *= require font-awesome
- *= require jasny-bootstrap
  */
 
 // Bootstrap v3.1.1 is already loaded in the WP theme
 //@import "bootstrap-sprockets";
 //@import "bootstrap";
 
+// Bootstrap Sass
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import "jasny-bootstrap";
+
+// Vendor Styles
+@import "font-awesome";
+
+// Application Styles
+@import "scaffolds";
+@import "home";
+@import "loans";
+@import "users";
+@import "media";
+@import "embedded";
+
```

Comment:
Why move all these to `@import`s? I was under the impression it was more efficient to use `require`s so that Rails compiles them all into one file.